### PR TITLE
Added a shell script to automate the whole installation process of DSR

### PR DESCRIPTION
**Pull request by manju838**
### Your checklist for this pull request

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your main!
- [ ] Make sure you are making a pull request against the **development branch** (left side). Also you should start *your branch* off *our development*.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] I have added necessary documentation (if appropriate)
- [x] It's small enought so they can be easily reviewed.
- [ ] I have CONSIDERED or ADDED a unit test if the PR resolves an issue.

## Types of changes
What types of changes does your code introduce to Robocomp?  

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Documentation Update
- [ ] Build related changes
- [ ] Other (please describe):

### Does this introduce a breaking change?

- [ ] Yes
- [x] No

### Does this close any currently open issues?
Issue ID: [ ]

### Description
Please describe your pull request.

===  
**Thank you!**

### CODE REVIEW

**Comment by orensbruli** [comment]
Hello! Thank you for your contribution.
Probably we could move to another directory.
And can you check if it can be merged with this existing one? https://github.com/robocomp/robocomp/blob/development/classes/dsr/dependencies.sh

**Comment by manju838** [comment]
@orensbruli Please check it now.I merged it with the dependencies.sh file.The initial code starts from line 41 onwards

File: DSR_Installscript.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+sudo git clone https://github.com/ryanhaining/cppitertools /usr/local/include/cppitertools
+cd /usr/local/include/cppitertools
+sudo mkdir build
+cd build
+sudo cmake ..
+sudo make install
+
+echo "Installing third-party software Fast-RTPS"
+
+mkdir software
```
Comment (orensbruli) [inline DSR_Installscript.sh]:
```suggestion
mkdir -p software
```
This would avoid mkdir to fail if software already exists.

**Comment by manju838** [comment]
@orensbruli Please check now,I made the suggested change.Thanks

**Pull request merged**